example/src/main/java/ExampleImGuizmo.java

```diff
@@ -0,0 +1,289 @@
+        r[7] = 0.0f;
+        r[8] = (right + left) / temp2;
+        r[9] = (top + bottom) / temp3;
+        r[10] = (-far - near) / temp4;
+        r[11] = -1.0f;
+        r[12] = 0.0f;
+        r[13] = 0.0f;
+        r[14] = (-temp * far) / temp4;
+        r[15] = 0.0f;
+        return r;
+    }
+
+    private static float[] cross(float[] a, float[] b) {
+        float[] r = new float[3];
+        r[0] = a[1] * b[2] - a[2] * b[1];
+        r[1] = a[2] * b[0] - a[0] * b[2];
+        r[2] = a[0] * b[1] - a[1] * b[0];
+        return r;
+    }
+
+    private static float dot(float[] a, float[] b) {
+        return a[0] * b[0] + a[1] * b[1] + a[2] * b[2];
+    }
+
+    private static float[] normalize(float[] a) {
+        float[] r = new float[3];
+        float il = (float) (1.f / (Math.sqrt(dot(a, a)) + FLT_EPSILON));
+        r[0] = a[0] * il;
+        r[1] = a[1] * il;
+        r[2] = a[2] * il;
+        return r;
+    }
+
+    private static void lookAt(float[] eye, float[] at, float[] up, float[] m16) {
+        float[] x;
+        float[] y;
+        float[] z;
+        float[] tmp = new float[3];
+
+        tmp[0] = eye[0] - at[0];
+        tmp[1] = eye[1] - at[1];
+        tmp[2] = eye[2] - at[2];
+        z = normalize(tmp);
+        y = normalize(up);
+
+        tmp = cross(y, z);
+        x = normalize(tmp);
+
+        tmp = cross(z, x);
+        y = normalize(tmp);
+
+        m16[0] = x[0];
+        m16[1] = y[0];
+        m16[2] = z[0];
+        m16[3] = 0.0f;
+        m16[4] = x[1];
+        m16[5] = y[1];
+        m16[6] = z[1];
+        m16[7] = 0.0f;
+        m16[8] = x[2];
+        m16[9] = y[2];
+        m16[10] = z[2];
+        m16[11] = 0.0f;
+        m16[12] = -dot(x, eye);
+        m16[13] = -dot(y, eye);
+        m16[14] = -dot(z, eye);
+        m16[15] = 1.0f;
+    }
+
+    public static void show() {
+
+        ImGui.setNextWindowSize(256, 256);
+        ImGui.begin("Editor");
+
+        if (firstFrame) {
+            float[] eye = {(float) (Math.cos(camYAngle) * Math.cos(camXAngle) * CAM_DISTANCE), (float) (Math.sin(camXAngle) * CAM_DISTANCE), (float) (Math.sin(camYAngle) * Math.cos(camXAngle) * CAM_DISTANCE)};
+            float[] at = {0.f, 0.f, 0.f};
+            float[] up = {0.f, 1.f, 0.f};
+            lookAt(eye, at, up, cameraView);
+        }
+
+        if (ImGuizmo.isUsing()) {
+            ImGui.text("Using gizmo");
+            ImGui.text(ImGuizmo.isOver() ? "Over a gizmo" : "");
+            ImGui.text(ImGuizmo.isOver(Operation.TRANSLATE) ? "Over translate gizmo" : "");
+            ImGui.text(ImGuizmo.isOver(Operation.ROTATE) ? "Over rotate gizmo" : "");
+            ImGui.text(ImGuizmo.isOver(Operation.SCALE) ? "Over scale gizmo" : "");
+        } else ImGui.text("Not using gizmo");
+        ImGui.separator();
+
+        editTransform();
+        ImGui.end();
+    }
+
+    private static void editTransform() {
+
+        if (ImGui.isKeyPressed(90)) currentGizmoOperation = Operation.TRANSLATE;
```

Comment:
Same thing with one-liners. Also, use `GLFW.GLFW_KEY_*` constants instead of raw numbers to express pressed key.